docs/design/2021-12-08-instance-scope.md

```diff
@@ -0,0 +1,224 @@
+# TiDB Design Documents
+
+- Author(s): [morgo](http://github.com/morgo)
+- Discussion PR: https://github.com/pingcap/tidb/pull/30558
+- Tracking Issue: https://github.com/pingcap/tidb/issues/30366
+
+## Table of Contents
+
+* [Introduction](#introduction)
+* [Motivation or Background](#motivation-or-background)
+* [Detailed Design](#detailed-design)
+* [Test Design](#test-design)
+* [Impacts & Risks](#impacts--risks)
+* [Investigation & Alternatives](#investigation--alternatives)
+* [Unresolved Questions](#unresolved-questions)
+
+## Introduction
+
+Currently, TiDB has two primary methods of configuration:
+
+- The configuration file (in `.toml` format) ([Docs](https://docs.pingcap.com/tidb/stable/tidb-configuration-file))
```

Comment:
```suggestion
- The configuration file (in `.toml` format) ([Docs](https://docs.pingcap.com/tidb/stable/tidb-configuration-file)) and `SHOW CONFIG`/`SET CONFIG`.
```

https://docs.pingcap.com/tidb/stable/dynamic-config

Reply:
`SET CONFIG` [doesn't work for TiDB](https://docs.pingcap.com/tidb/stable/dynamic-config#modify-tidb-configuration-online) though. And `SHOW CONFIG` is only a summary of what is set in the configuration file, so I would not class it as a 'method of configuration'.